dotcom-rendering/src/components/MoreGalleries.tsx

```diff
@@ -0,0 +1,269 @@
+		showByline: trail.showByline,
+		showQuotedHeadline: trail.showQuotedHeadline,
+		webPublicationDate: trail.webPublicationDate,
+		kickerText: trail.kickerText,
+		showPulsingDot: false,
+		showClock: false,
+		image: trail.image,
+		isCrossword: trail.isCrossword,
+		starRating: trail.starRating,
+		dataLinkName: trail.dataLinkName,
+		snapData: trail.snapData,
+		discussionApiUrl,
+		discussionId: trail.discussionId,
+		avatarUrl: trail.avatarUrl,
+		mainMedia: trail.mainMedia,
+		isExternalLink: false,
+		branding: trail.branding,
+		absoluteServerTimes,
+		imageLoading: 'lazy',
+		trailText: trail.trailText,
+		showAge: false,
+		containerType: 'more-galleries',
+		showTopBarDesktop: false,
+		showTopBarMobile: false,
+		aspectRatio: '5:4',
+	};
+	return defaultProps;
+};
+
+export const MoreGalleries = (props: Props) => {
+	const [firstTrail, ...standardCards] = props.trails;
+	if (!firstTrail) return null;
+
+	const defaultProps = getDefaultCardProps(
+		firstTrail,
+		props.absoluteServerTimes,
+		props.discussionApiUrl,
+	);
+
+	return (
+		<Section
+			fullWidth={true}
+			borderColour={palette('--onward-content-border')}
+			backgroundColour={palette('--onward-background')}
+			showTopBorder={false}
+		>
+			<div
+				css={wrapperStyle}
+				data-link-name={formatAttrString(props.heading)}
+			>
+				<LeftColumn
+					size={'compact'}
+					borderColour={palette('--onward-content-border')}
+					hasPageSkin={false}
+				>
+					<Title title={props.heading} url={props.url} />
+				</LeftColumn>
+
+				<div
+					css={containerStyles}
+					data-component={props.onwardsSource}
+					data-link={formatAttrString(props.heading)}
+				>
+					<Hide when="above" breakpoint="leftCol">
+						<Title title={props.heading} url={props.url} />
+					</Hide>
+
+					<MoreGalleriesSplashCard defaultProps={defaultProps} />
+					<Hide when="below" breakpoint="tablet">
+						<StraightLines
+							count={1}
+							color={palette('--onward-content-border')}
+						/>
+					</Hide>
+
+					<ul css={standardCardsListStyles}>
+						{standardCards.map((trail) => (
+							<li key={trail.url} css={standardCardStyles}>
+								{Card({
+									...getDefaultCardProps(
+										trail,
+										props.absoluteServerTimes,
+										props.discussionApiUrl,
+									),
+									mediaSize: 'medium',
+								})}
+							</li>
+						))}
+					</ul>
+				</div>
+			</div>
+		</Section>
+	);
+};
+
+const MoreGalleriesSplashCard = ({
+	defaultProps,
+}: {
+	defaultProps: CardProps;
+}) => {
+	const cardProps: Partial<CardProps> = {
+		headlineSizes: {
+			desktop: 'medium',
+			tablet: 'medium',
+			mobile: 'medium',
+		},
+		mediaPositionOnDesktop: 'right',
+		mediaPositionOnMobile: 'top',
+		mediaSize: 'medium',
+		isFlexSplash: true,
+	};
+	return (
+		<div
+			css={css`
+				margin-bottom: ${space[6]}px;
+				background-color: ${palette('--onward-card-background')};
+				padding: ${space[2]}px;
+			`}
+		>
+			{Card({ ...defaultProps, ...cardProps })}
```

Comment:
As mentioned above, it's best to use `createElement` (JSX) to render React components:

```suggestion
			<Card {...defaultProps} {...cardProps} />
```